pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -185,8 +199,9 @@ type OFBridge struct {
 	// connected is an internal channel to notify if connected to the OFSwitch or not. It is used only in Connect method.
 	connected chan bool
 	// pktConsumers is a map from PacketIn reason to the channel that is used to publish the PacketIn message.
-	pktConsumers      sync.Map
-	multipartReplyChs map[uint32]chan *openflow13.MultipartReply
+	pktConsumers         sync.Map
+	multipartReplyChs    map[uint32]chan *openflow15.MultipartReply
+	tunMetadataLengthMap map[uint16]uint8
```

Comment:
Could you add some comments about this?

Reply:
Another question: is it used anywhere? I didn't see any code read its value.

Reply:
This is used in function `AddTLVMap` which function is already existing before this change. The function was originally designed for the scenario to leverage Geneve header to transport some customized information, e.g., in TraceFlow or streched NP features.  But it seems they are not consuming this solution now. @tnqn do you think we should remove function `AddTLVMap` together?